spec/src/main/java/io/a2a/spec/ListTasksRequest.java

```diff
@@ -4,25 +4,16 @@
 
 import java.util.UUID;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
 import io.a2a.util.Assert;
 
 /**
  * A list tasks request.
  */
-@JsonInclude(JsonInclude.Include.NON_ABSENT)
-@JsonIgnoreProperties(ignoreUnknown = true)
 public final class ListTasksRequest extends NonStreamingJSONRPCRequest<ListTasksParams> {
 
     public static final String METHOD = "ListTask";
 
-    @JsonCreator
-    public ListTasksRequest(@JsonProperty("jsonrpc") String jsonrpc, @JsonProperty("id") Object id,
-                            @JsonProperty("method") String method, @JsonProperty("params") ListTasksParams params) {
+    public ListTasksRequest(String jsonrpc, Object id, String method, ListTasksParams params) {
         if (jsonrpc == null || jsonrpc.isEmpty()) {
```

Comment:
nitpick, the validation differs from https://github.com/a2aproject/a2a-java/pull/514/files#diff-1a074a4d32c70956ec3ec7aae747489990148b90c64d043c0e35f21c2e1b8ef5R26 that checks the version value too